source/command-monitoring/tests/find.yml

```diff
@@ -195,8 +195,9 @@ tests:
               - { $numberLong: "42" }
           command_name: "killCursors"
   -
-    description: "A successful find event with a getmore and the server kills the cursor"
+    description: "A successful find event with a getmore and the server kills the cursor (<= 4.4)"
     ignore_if_server_version_less_than: "3.1"
+    ignore_if_server_version_greater_than: "4.4"
```

Comment:
For the benefit of other reviewers, this covers all 4.4.x minor versions (per [README](https://github.com/mongodb/specifications/blob/master/source/command-monitoring/tests/README.rst#ignoring-tests-based-on-server-version-or-topology-type)). It's been a while since I looked at this test format so I wanted to double-check.